### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by jhutchings1**
Creating an org-wide security policy based upon the Bug Bounty program definition. This will show up in any public repositories in the `github` organization.

### CODE REVIEW

**Review by benbalter — APPROVED** [review]

**Comment by shawndavenport** [comment]
>  This will show up in any public repositories in the `github` organization.

Does that exclude forks?

/cc @github/appsec

**Review by philipturnbull — APPROVED** [review]
Looks like a great start 👍 I will follow up next week with a separate PR to clarify some points around  forked repos, archived repos, scope, etc

**Comment by jhutchings1** [comment]
Does anyone with `write` access want to merge this? I don't have permissions.

**Comment by createdbyhim63** [comment]
Pull requests

**Comment by lyric-k** [comment]
^^Thanks!!!!!

**Review by lyric-k — APPROVED** [review]
Thanks🤗

**Comment by 0867085052** [comment]
Hi

**Review by luismedicina23 — COMMENTED** [review]
SECURITY.md

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# GitHub Security Policy
```
Comment (luismedicina23) [inline SECURITY.md:1]: @jhutchings1

**Pull request merged**